.golangci.yml

```diff
@@ -38,6 +38,7 @@ linters:
     - contextcheck
     - cyclop
     - exhaustivestruct
+    - exhaustruct
```

Comment:
why are we disabling this one. it's generally good practice to force all members on struct declarations?